src/collectd.conf.pod

```diff
@@ -3485,6 +3485,21 @@ used as the type-instance.
 If I<Name> is supplied, it will be used as the type-instance instead of the
 comment or the number.
 
+  <Plugin iptables>
+    Chain "mangle" "/^custom_chain/"
+    Chain6 "magle" "custom_chain" "comment_name"
```

Comment:
Typo? ("magle" instead of "mangle")

Reply:
Yes its typo. I will raise another pull request for this change.